source/Panel.cpp

```diff
@@ -23,8 +23,8 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Point.h"
 #include "Preferences.h"
 #include "Screen.h"
-#include "text/Format.h"
 #include "UI.h"
+#include "text/Format.h"
```

Comment:
If this were a good change, then I suggest to do this in a separate PR. This PR is already quite large so I prefer to keep it focused without moving includes around.

Reply:
That change was accidental, when I was trying to get rid of a warning from the coding style checker. I will revert it.